library/std/src/num/f64.rs

```diff
@@ -749,12 +749,12 @@ impl f64 {
     /// # Examples
     ///
     /// ```
-    /// let f = std::f64::consts::FRAC_PI_2;
+    /// let f = std::f64::consts::FRAC_PI_4;
```

Comment:
Whenever you change the test values (and not just the tolerance), please ensure you do this consistently for all four `f*.rs` files.